Makefile

```diff
@@ -154,11 +154,11 @@ multimod-verify: | $(MULTIMOD)
 	$(MULTIMOD) verify
 
 COREPATH ?= "../opentelemetry-collector-contrib"
-OPTIONAL_CORE_PATH ?="../opentelemetry-collector"
+SKIP_GO_MOD_TIDY ?= "true"
 .PHONY: multimod-sync-core
 multimod-sync-core: multimod-verify
-	@[ ! -d $COREPATH ] || ( echo ">> Path to core repository must be set in COREPATH and must exist"; exit 1 )
-	$(MULTIMOD) sync -a -o ${COREPATH} -o ${OPTIONAL_CORE_PATH}
+	@[ ! -d COREPATH ] || ( echo ">> Path to core repository must be set in COREPATH and must exist"; exit 1 )
+	$(MULTIMOD) sync -a -o ${COREPATH} --skip-go-mod-tidy ${SKIP_GO_MOD_TIDY}
```

Comment:
I don't think we should skip go mod tidy.

Reply:
One of the reasons why I added the ```SKIP_GO_MOD_TIDY_PATH``` flag is: in case of the upstream repo changed the components, the ```go mod tidy``` does not allow us to update the dependencies. But mostly the ```sync-core``` main's function was to update dependencies, so that's why I want to keep that flow.
![image](https://user-images.githubusercontent.com/91758108/142483795-66b5136f-efbe-4b5c-91ab-a82f1787c707.png)